src/platform/Linux/Logging.cpp

```diff
@@ -43,7 +43,6 @@ void LogV(const char * module, uint8_t category, const char * msg, va_list v)
     // indicate the error occurred during getting time.
     gettimeofday(&tv, nullptr);
 
-#if !CHIP_USE_PW_LOGGING
```

Comment:
Do we still need the `pw_log/log.h` include above?

Reply:
Removed all references.